libs/sdk-embedding/src/iframe/kd.ts

```diff
@@ -1272,4 +1282,45 @@ export namespace EmbeddedKpiDashboard {
         GdcKdEventType.SetFilterParentsFailed,
         ISetFilterParentsFailedDataBody
     >;
+
+    /**
+     * Open delete dashboard dialog, user will be able to delete the current existed dashboard
+     *
+     * Contract:
+     *
+     * if KD is currently editing dashboard, this command will try to open the dialog to delete the current existed dashboard,
+     *      on success DeleteDashboardDialogOpened will be posted
+     * commandFailed will be posted when:
+     *      KD is currently viewing dashboard or
+     *      No dashboard showing or
+     *      Donnot have permission delete existing object or
```

Comment:
Typo: Donnot. I would suggest rewording this to something like:
The current user does not have the permission to delete existing objects